Importing a single codec when using a browser build

From node.js it is possible to import just a single codec, but this does not seem to work from a browser-targeted build using typescript/webpack.  Typescript gives the error:

```typescript
import {Blosc} from 'numcodecs/blosc';
```

```
Cannot find module 'numcodecs/blosc' or its corresponding type declarations.
```

Using


```typescript
import {Blosc} from 'numcodecs';
```

works but also pulls in pako, which increases the bundle size a fair bit.

--- a/package.json
+++ b/package.json
@@ -24,6 +24,7 @@
       "require": "./dist/blosc.cjs"
     }
   },
+  "sideEffects": false,
   "repository": {
     "type": "git",
     "url": "https://github.com/manzt/numcodecs.js.git"

